debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SnapshotIT.java

```diff
@@ -311,7 +311,7 @@ public void takeSchemaOnlySnapshotAndSendHeartbeat() throws Exception {
 
     @Test
     @FixFor("DBZ-1067")
-    public void blacklistColumn() throws Exception {
+    public void testBlacklistColumn() throws Exception {
```

Comment:
Why did you change the name? We don't add the "test" really (although it's handled a bit incosistently; my personal preference is something like "blacklistedColumnShouldNotShowUpInChangeEvent", i.e. describing the expectation/intention behind the test.

Reply:
we always have "should" everywhere, LOL. just personally I find it's easier to see if something is actually a test (when my brain fades out the "@Test" annotation) when it starts with test/should prefix. and I think there are prefixes everywhere for actual test cases and no prefix for helper methods. just thought it is off (personal OCD mind)

Reply:
Well, the annotation really _is_ the hint that something is a test method. No need to duplicate that information into the name.